packages/web-app-files/src/components/SideBar/Actions/SpaceActions.vue

```diff
@@ -113,11 +113,21 @@ export default defineComponent({
 
 #oc-spaces-actions-sidebar {
   > li a,
-  > li a:hover {
+  > li a:hover { /* required? file actions are buttons not links. Looks like legacy */
     color: var(--oc-color-swatch-passive-default);
     display: inline-flex;
     gap: 10px;
     vertical-align: top;
   }
+
+  > li button { /* Overwrites button class oc-button-justify-content-center, but how and where to change this class? */
+    justify-content: left;
+    width: 100%;
```

Comment:
for width we have the CSS class oc-width-1-1, I prefer those classes over css.
You might also check if oc-flex-left or oc-text-left migt be sufficient.